deploy/goerli/deploy_goerli_MarginAccountFactory.ts

```diff
@@ -0,0 +1,39 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+
+/*///////////////////////////////////////////////////////////////
+                TEST DEPLOYMENT ON OPTIMISTIC GOERLI
+///////////////////////////////////////////////////////////////*/
+
+const deployMarginAccountFactoryOnGoerli: DeployFunction = async function (
+    hre: HardhatRuntimeEnvironment
+) {
+    const { deployments, getNamedAccounts } = hre;
+    const { deploy } = deployments;
+    const { deployer } = await getNamedAccounts();
+
+    // constructor params
+    const version = "1.0.0";
+    const marginAsset = "0xeBaEAAD9236615542844adC5c149F86C36aD1136"; // ProxyERC20sUSD on Goerli
+    const addressResolver = "0x9Fc84992dF5496797784374B810E04238728743d"; // ReadProxyAddressResolver on Goerli
+    const marginBaseSettings = await deployments.get("MarginBaseSettings"); // Settings on Goerli
+
+    // @todo update for Goerli (might not change...)
+    const ops = "0xB3f5503f93d5Ef84b06993a1975B9D21B962892F"; // Ops on Kovan
```

Comment:
Heads up they are live, just got the address from them: `0x255F82563b5973264e89526345EcEa766DB3baB2`

Reply:
Nice. I asked them in telegram but they never replied

Reply:
rip